Tasks/Common/nuget-task-common/NuGetToolGetter.ts

```diff
@@ -23,7 +23,15 @@ enum NuGetReleaseStage
 const NUGET_TOOL_NAME: string = 'NuGet';
 const NUGET_EXE_FILENAME: string = 'nuget.exe';
 
-export async function getNuGet(versionSpec: string, checkLatest: boolean): Promise<string> {
+export const NUGET_EXE_TOOL_PATH_ENV_VAR: string = 'NuGetExeToolPath';
+
+export async function getNuGet(versionSpec?: string, checkLatest?: boolean, addNuGetToPath?: boolean): Promise<string> {
+    if (!versionSpec)
+    {
+        versionSpec = "4.0.0";
```

Comment:
This might work, given the way that snapshots are created for previous major versions of the task, but having the default NuGet version specified in nuget-task-common instead of in each task might make major version updates awkward.

For example, lets say that we're on v1 of all NuGet tasks, and they use NuGet v 4.0.0 as default.
If we need a new version of the NuGetInstaller task that uses NuGet v 5.0.0 and we bump its major version... if we update the default version here to 5.0.0 we are now forced to also bump the major version of every other nuget task so they don't also pick up this change.

Maybe that's ok, but it seems that the design would be more solid if the task supplies its default version to the common lib.